components/Layout/HeadingSlackCTA/HeadingSlackCTA.module.scss

```diff
@@ -0,0 +1,39 @@
+.container {
+    display: flex;
+    align-items: center;
+    position: absolute;
+    top: 1rem;
+    color: inherit;
+
+    @include media-query(large) {
+        right: 0;
+    }
+
+    .logo {
+        width: 31px;
+        height: 31px;
+    }
+
+    .cta {
+        @include font-size(xs);
+        display: flex;
+        flex-direction: row;
+        margin-left: 0.5rem;
+        line-height: 1.2;
+        font-weight: $weight--extrabold;
+
+        @include media-query(large) {
+            flex-direction: column;
+        }
+
+        &__tag {
+            color: var(--color--accent);
+            font-weight: inherit;
+            margin: 0 0.1rem;
```

Comment:
```suggestion
            margin: 0 0.2rem;
```
Just a tiny bit more margin here to give a proper space between the text

<img width="310" alt="image" src="https://user-images.githubusercontent.com/18164832/155693268-66bb9845-1d8e-4610-987c-3bf5698ecca8.png">

Reply:
Added